package.json

```diff
@@ -1,13 +1,16 @@
 {
   "license": "MIT",
   "scripts": {
-    "start": "node app.js",
+    "start": "npm-run-all --parallel watch:app watch:scss",
```

Comment:
I'm not a fan of having `start` start in dev mode. I'd prefer leaving `start` as was, and add a `start:dev` script for dev mode.